WordPress/src/main/java/org/wordpress/android/ui/accounts/signup/SignupEpilogueFragment.java

```diff
@@ -591,6 +591,7 @@ public void onLoaded() {
     }
 
     private void populateViews() {
+        AnalyticsTracker.track(AnalyticsTracker.Stat.CREATED_ACCOUNT);
```

Comment:
Hey @theck13 , wdyt about leaving the track trigger outside the `populateViews()` method to keep it clear of side effect like that (it's a UI related method anyway) and instead, just do the trigger at the 2 method call sites?

Alternatively, and since I'm actually a bit worried triggering this track so late in the process, wdyt about adding the track closer to the places we have this info the earliest? Those places would be the in WPMainActivity for magiclink sign and in SignupGoogleFragment as you already have it in this PR?

Reply:
I moved the trigger to the signup epilogue launch in e1a88d6.